src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
+const minDaysForDiscount = 3;
+const maxDaysForDiscount = 7;
+const baseDiscountAmount = 20;
+const largeDiscountAmount = 50;
+const dailyRentalRate = 40;
```

Comment:
You need to move these variables inside the function, as they are only used within the function